test/fatptr/lit.cfg

```diff
@@ -0,0 +1,67 @@
+# -*- Python -*-
```

Comment:
We will now have too many tests with similar names. Xiang's tests are under `fat_ptr`. Need to either unify, or rename to be more distinct.

Reply:
Renamed to fat_ptr_symbolic...can unify later as needed.